package.json

```diff
@@ -47,7 +47,7 @@
     "time-grunt": "^0.3.1",
     "uuid": "1.4.x",
     "web-literacy-client": "1.1.1",
-    "webmaker-auth": "1.0.2",
+    "webmaker-auth": "https://github.com/mozilla/webmaker-auth/tarball/v1.0.3",
```

Comment:
let's set this to 1.0.3, because the fact that it's not on npm yet is our mistake. @cadecairos or @jbuck can help us fix that one =)